tests/test_flow/test_protocols.py

```diff
@@ -381,8 +381,12 @@ def raw_df():
     def counts_df(raw_df):
         return raw_df.groupby(["continent", "country"]).size().to_frame("count")
 
-    with pytest.raises(UnsupportedSerializedValueError):
+    try:
         builder.build().get("counts_df")
```

Comment:
For Dask >= 2.22.0, can you add an assertion here that checks a value from `counts_df`?

Reply:
So, it turns out that Dask's new behavior here is [non-deterministic](https://github.com/dask/dask/issues/6484), so I'll have to add a somewhat vague check. Oh well.

Reply:
Till you get an answer from dask developers, I think it's okay to merge your original change (without any value checks) and the other PR that depends on it, this blocks a lot of your other work. You can open another PR later once you hear back from them.

Reply:
I agree. I modified the PR to add a minimal check in the test, but I don't think it's worth getting blocked any further on this.

BTW, based on the discussion so far in the linked thread, it sounds like it make more sense to use `fastparquet` for Dask DataFrames, but that can happen in a different PR.